training/src/main/java/com/wolox/training/dto/BookInputDto.java

```diff
@@ -0,0 +1,21 @@
+package com.wolox.training.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class BookInputDto {
```

Comment:
This class is not necessary, Camel can transform the Book instance into JSON and vice versa